x-pack/plugins/observability_solution/observability_ai_assistant/README.md

```diff
@@ -6,35 +6,27 @@ This document gives an overview of the features of the Observability AI Assistan
 
 #### **1.1. Configuration**
 
-Users can connect to an LLM using [connectors](https://www.elastic.co/guide/en/kibana/current/action-types.html) - specifically the [OpenAI connector](https://www.elastic.co/guide/en/kibana/current/openai-action-type.html), which currently supports both OpenAI and Azure OpenAI as providers. The connector is Enterprise-only. Users can also leverage [preconfigured connectors](https://www.elastic.co/guide/en/kibana/current/pre-configured-connectors.html), in which case the following should be added to `kibana.yml`:
+Users can connect to an LLM using [connectors](https://www.elastic.co/guide/en/kibana/current/action-types.html) - specifically the [OpenAI connector](https://www.elastic.co/guide/en/kibana/current/openai-action-type.html) or the [Bedrock connector](https://www.elastic.co/guide/en/kibana/current/bedrock-action-type.html), which currently supports both OpenAI and Azure OpenAI as providers. The connector is Enterprise-only. Users can also leverage [preconfigured connectors](https://www.elastic.co/guide/en/kibana/current/pre-configured-connectors.html), in which case the following should be added to `kibana.yml`:
```

Comment:
If the target audience is internal folks, should we add a link to the openai credentials (the alias to the secure pastebin)?

Reply:
Hmmm, I'm not sure, for two reasons: 
- we'll switch to creds for the Obs org at some point. I'm not sure if we should advertise these credentials to everyone, because it will make it harder to track costs.
- it doesn't feel right to leave link to creds here, in a public repo? I know that they are secured now, but that's no guarantee, also not sure if this for instance will cause some bots to start scraping the URL for whatever reason.